nixos/tests/all-tests.nix

```diff
@@ -75,6 +75,7 @@ in {
   airsonic = handleTest ./airsonic.nix {};
   allTerminfo = handleTest ./all-terminfo.nix {};
   amazon-init-shell = handleTest ./amazon-init-shell.nix {};
+  amdvlk = handleTest ./amdvlk.nix {};
```

Comment:
This is better but I would still not include these tests in `all-tests.nix` since we include this in release.nix and therefore built them with hydra. Also those tests are not actually nixos vm tests like the others which is a bit confusing.
I have two solutions for that:

Put all those impure tests in `./pkgs/test` or just put them into the package directory along side the `default.nix` (my preference actually).